### ISSUE TITLE
Add basic auth

### ISSUE THREAD

**Issue by aleksandr-smechov** [Wordcab/wordcab-transcribe#41]
Add basic auth

**Comment by chainyo** [comment]
Is this issue resolved by #50, or should we also implement a more beefy auth tool (like Bearer token auth) for the other endpoints?

**Comment by aleksandr-smechov** [comment]
Yes let's get the Bearer authentication in place for the other endpoints.

**Comment by chainyo** [comment]
> Yes let's get the Bearer authentication in place for the other endpoints.

Great, I will add it.

**Comment by aleksandr-smechov** [comment]
Thanks Thomas
